editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4082,6 +4091,8 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	show_rulers = true;
 	show_guides = true;
 	zoom = 1;
+	view_offset = Point2(-150 - RULER_WIDTH, -150 - RULER_WIDTH);
```

Comment:
Can you add an explanatory comment for the magic numbers?
I was going more for a rectangular offset to somewhat mimic the aspect ratio (ca. 16:9/16:10), but I'm fine with a square if you think it looks good.